projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -235,24 +236,7 @@ export class ClrDatagridWidgetObject<T> extends BaseWidgetObject<T> {
         return this.locatorDriver.get(Css.COLUMN).get(Css.FILTER).get(Css.FILTER_TOGGLE).unwrap();
     }
 
-    /**
-     * Find a clr-checkbox-wrapper
-     */
-    getCheckboxWrapper(): T {
-        return this.locatorDriver.get(Css.CHECKBOX_WRAPPER).unwrap();
-    }
-
-    /**
-     * Find a clr-radio-wrapper
-     */
-    getRadioWrapper(): T {
-        return this.locatorDriver.get(Css.RADIO_WRAPPER).unwrap();
-    }
-
-    /**
-     * Unwrap the clr-datagrid
-     */
-    unwrap(): T {
+    get component(): T {
```

Comment:
These methods make it seem like we're getting the component instance, can we rename these to refer to the name of the component? So this one would be `get clrDatagrid`?